README.md

```diff
@@ -71,3 +71,13 @@ class with child classes for the various permutations of TRACK3R
 * TRACK3RWithBallShooter
 * TRACK3RWithClaw
 * TRACK3RWithSpinner
+
+## Differences between Home and Education sets
```

Comment:
What would you think about just focussing on "more demo programs" and omitting the mentioning of the comparison of the Home Edition and the Education Edition? I don't think that users or developers coming to this README primarily care about the difference between the two hardware sets. They may care about "more robot programs", though!

Hence, the paragraph may look like instead:
> ## More Robot Programs
> 
> The [LEGO Mindstorms EV3 Comparison project](https://github.com/bittner/lego-mindstorms-ev3-comparison) sports a few other robot programs. Its [comparison table](https://github.com/bittner/lego-mindstorms-ev3-comparison#inspiration-for-lego-ev3-robots) also lists programs created for other platforms than ev3dev.

What do you guys think?